### ISSUE TITLE
Choosing the output neuron

### ISSUE THREAD

**Issue by Gijom** [tensorflow/playground#29]
Wonderful !

Would it be possible to clarify the output neuron. I can imagine it's activation function is a sigmoid. What kind of loss is employed ? Would it be possible to propose different loss functions ?
